packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -263,7 +281,15 @@ class ChartController extends DisposableController
 
   double get fixedMaxY => _fixedMaxY;
 
-  String title;
+  String get title => _title;
```

Comment:
by convention place the field in between the getter and setter.

Reply:
Done.